src/mame/casio/cz230s.cpp

```diff
@@ -0,0 +1,693 @@
+// license: BSD-3-Clause
+// copyright-holders: Devin Acker
+/***************************************************************************
+    Casio CZ-230S digital synthesizer and SZ-1 MIDI sequencer
+
+    Misc. stuff:
+    Both of these devices have a way of loading and running external code.
+    - CZ-230S:
+      Hold "portamento speed", "value up", and "value down" together on boot. This will cause the
+      LCD to display "L-", as when loading from tape. At this point, the unit will try to load
+      $700 bytes over MIDI/serial to address $3800 and then jump to it.
+    - SZ-1:
+      While not recording or playing, pressing the Rest + Dot + Triplet buttons at the same time will
+      cause the firmware to check for a JMP instruction (54) at the first byte of cartridge memory
+      ($e000), and execute it if there is one.
+
+    TODO: auto power off. Even after activating this, both units still continue executing as normal
+    (and the power switch itself is not connected to the CPU, unlike on the CZ-101/1000)
+
+***************************************************************************/
+
+#include "emu.h"
+
+#include "ra3.h"
+#include "bus/midi/midiinport.h"
+#include "bus/midi/midioutport.h"
+#include "cpu/upd7810/upd7811.h"
+#include "imagedev/cassette.h"
+#include "machine/clock.h"
+#include "machine/nvram.h"
+#include "sound/beep.h"
+#include "sound/upd933.h"
+#include "sound/upd934g.h"
+#include "video/mn1252.h"
+
+#include "screen.h"
+#include "speaker.h"
```

Comment:
Can you put a blank line after `#include "ra3.h"` to visually separate local project headers from the src/devices headers?